.github/workflows/bench.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           name: Arm Cortex-A55 (Snapdragon 888) benchmarks
           bench_pmu: PERF
           archflags: "-mcpu=cortex-a55 -march=armv8.2-a"
-          cflags: "-flto -static -DFORCE_AARCH64 -DFIPS202_NATIVE_PROFILE=\"aarch64/profiles/cortex_a55.h\""
+          cflags: "-flto -static -DFORCE_AARCH64 -DMLKEM_NATIVE_FIPS202_BACKEND=\\\\\\\"aarch64/cortex_a55.h\\\\\\\""
```

Comment:
Not sure if I should cry or laugh. The hour of trial-and-error before I got this right certainly made me want to cry.

Reply:
This is so that `MLKEM_NATIVE_FIPS202_BACKEND` evaluates to a _string_ in C, and we can set `#define MLKEM_NATIVE_FIPS202_BACKEND "foo.h"` in the `config.h`. Previously, `FIPS202_NATIVE_PROFILE` played the role of `MLKEM_NATIVE_FIPS202_BACKEND` but was stringified within C, so one had to write `#define MLKEM_NATIVE_FIPS202_BACKEND foo.h`, which arguably is unnatural.